.env.example

```diff
@@ -0,0 +1,10 @@
+####################
+### EXAMPLE FILE ###
+####################
+
+# Do NOT edit this file unless adding new env variables
+# Instead, copy it, remove the .example extension and provide your custom values
+
+WINDUP_PATH=path/to/windup-cli
+OUTPUT_PATH=path/to/export/the/report/in/
```

Comment:
and this to "report_output_path"

Reply:
Solved in a494b7ee79d5cdf745c75b942d9b094ba1f6dfcf.